### ISSUE TITLE
GitHub

### ISSUE THREAD

**Issue by StanJr1** [patanjalikr13/Android-Undetectable-Spy-App#4]
Hello, kindly help in the installation of the undetectable spy software

**Comment by EhtishamSabir** [comment]
Yes please update the app bro

**Comment by patanjalikr13** [comment]
Hi guys, app is working fine. Lemme which part you are stuck on ?

**Comment by EhtishamSabir** [comment]
Can you try it on android 7+? because I have been trying to build it's giving errors.

**Comment by kumarutsav23** [comment]
> 
> 
> Hi guys, app is working fine. Lemme which part you are stuck on ?

I've done everything, provided accessibility and even auto start permission. But nothing is showing in live database
![image](https://user-images.githubusercontent.com/71973933/94991128-84727f80-059e-11eb-9818-9a8a596f56db.png)


Please help me out fast

**Comment by sunny269** [comment]
does the app contain admin and user side

**Comment by patanjalik** [comment]
No, app doesn't not contain user and admin side. This is mainly for users familiar with little bit of programming.

@EhtishamSabir lemme know which error.
